frontend/pages/hosts/details/cards/HostSummary/HostSummary.tsx

```diff
@@ -317,6 +317,39 @@ const HostSummary = ({
     );
   };
 
+  const renderAgentSummary = () => {
+    if (platform === "chrome") {
+      return <DataSet title="Agent" value={summaryData.osquery_version} />;
+    }
+    if (summaryData.orbit_version) {
+      return (
+        <DataSet
+          title="Agent"
+          value={
+            <TooltipWrapper
+              tipContent={
+                <>
+                  osquery: {summaryData.osquery_version}
+                  <br />
+                  Orbit: {summaryData.orbit_version}
+                  {summaryData.fleet_desktop_version && (
+                    <>
+                      <br />
+                      Fleet Desktop: {summaryData.fleet_desktop_version}
```

Comment:
I see this is "Fleet Desktop" in Figma, but do we want "Fleet desktop" per [sentence casing](https://fleetdm.com/handbook/company/why-this-way#why-does-fleet-use-sentence-case) @rachaelshaw?

Reply:
Thanks for all the thorough reviews Jacob!!

Looking at other instances in our repo, they're all proper titled Fleet Desktop -- I'm going to merge but can update if needed.